packages/composer/test/graphql/watch.test.js

```diff
@@ -150,16 +96,12 @@ test('composer should restart and update schema if one of the services shuts dow
     }
   )
 
-  await composer.start()
-  const restart = eventToPromise(composer.addOnRestartHook, RESTART_TIMEOUT)
-
-  assert.equal(composer.graphqlSupergraph.sdl, supergraph1)
+  await runtime.start()
+  await sleep(REFRESH_TIMEOUT * 2)
 
   await graphql1.close()
-  await restart
 
-  assert.equal(composer.restarted, true, 'expected composer to restart')
-  assert.equal(composer.graphqlSupergraph.sdl, supergraph2)
```

Comment:
this assertion is important

Reply:
I restored in a different way but I now check schemas.